commands/channelargs.go

```diff
@@ -56,3 +59,64 @@ func getChannelFromChannelArg(c client.Client, channelArg string) *model.Channel
 
 	return channel
 }
+
+// getChannelsFromArgs obtains channels by the `channelArgs` parameter. It can return channels and errors
+// at the same time
+//nolint:golint,unused
+func getChannelsFromArgs(c client.Client, channelArgs []string) ([]*model.Channel, error) {
+	var channels []*model.Channel
+	var result *multierror.Error
+	for _, channelArg := range channelArgs {
+		channel, err := getChannelFromArg(c, channelArg)
+		if err != nil {
+			result = multierror.Append(result, err)
+		} else {
+			channels = append(channels, channel)
+		}
+	}
+	return channels, result.ErrorOrNil()
```

Comment:
We can simplify this a bit and save some indentation for the happy path. This applies to other similar occurrences.
```suggestion
	for _, channelArg := range channelArgs {
		channel, err := getChannelFromArg(c, channelArg)
		if err != nil {
			result = multierror.Append(result, err)
			continue
		}
		channels = append(channels, channel)
	}
	return channels, result.ErrorOrNil()
```